MvvmCross/Droid/Droid/Platform/MvxAndroidSetup.cs

```diff
@@ -158,7 +159,7 @@ protected virtual void RegisterBindingBuilderCallbacks()
             Mvx.CallbackWhenRegistered<IMvxNamespaceListViewTypeResolver>(FillNamespaceListViewTypeResolver);
         }
 
-        protected virtual MvxAndroidBindingBuilder CreateBindingBuilder()
+        protected virtual MvxBindingBuilder CreateBindingBuilder()
```

Comment:
So we can use the overloaded class without reimplementing the InitializeBindingBuilder.  Look at the Android Setup.cs